docs/view/Skeleton/README.md

```diff
@@ -0,0 +1,142 @@
+# Skeleton
+
+## Installation
+
+Backpack Android is available through [Maven Central](https://search.maven.org/artifact/net.skyscanner.backpack/backpack-android). Check the main [Readme](https://github.com/skyscanner/backpack-android#installation) for a complete installation guide.
+
+## Usage
```

Comment:
same here - make it clearer that it should generally be used within the shimmer overlay

Reply:
ok